tests/test_cache_blocking.py

```diff
@@ -38,6 +39,7 @@ def test_cache_blocking_edge_cases(self, shape, time_order,
         self.cache_blocking_test(shape, time_order, spc_border, cache_blocking)
 
     def cache_blocking_test(self, shape, time_order, spc_border, cache_blocking):
+        clear_cache()
```

Comment:
Oh wow, confirmed, this does look positively evil. This test is very high on the consolidation list for the actual operator flip though, so we will probably deal with it then. I'm happy to go ahead then.